pkg/action/install.go

```diff
@@ -105,15 +106,27 @@ type Install struct {
 
 // ChartPathOptions captures common options used for controlling chart paths
 type ChartPathOptions struct {
-	CaFile   string // --ca-file
-	CertFile string // --cert-file
-	KeyFile  string // --key-file
-	Keyring  string // --keyring
-	Password string // --password
-	RepoURL  string // --repo
-	Username string // --username
-	Verify   bool   // --verify
-	Version  string // --version
+	CaFile           string // --ca-file
+	CertFile         string // --cert-file
+	KeyFile          string // --key-file
+	Keyring          string // --keyring
+	Password         string // --password
+	RepoURL          string // --repo
+	Username         string // --username
+	Verify           bool   // --verify
+	Version          string // --version
+	UseInMemoryCache bool   // --use-inmemory-cache
```

Comment:
To be more generic, how about naming this `readOnlyFS`?

Reply:
Updated, this is a great idea !